base_tier_validation/models/tier_review.py

```diff
@@ -20,22 +21,40 @@ class TierReview(models.Model):
     model = fields.Char(string="Related Document Model", index=True)
     res_id = fields.Integer(string="Related Document ID", index=True)
     definition_id = fields.Many2one(comodel_name="tier.definition")
-    review_type = fields.Selection(related="definition_id.review_type", readonly=True)
-    reviewer_id = fields.Many2one(related="definition_id.reviewer_id", readonly=True)
+    review_type = fields.Selection(
+        selection=[
+            ("individual", "Specific user"),
+            ("group", "Any user in a specific group."),
+        ],
+        compute="_compute_definition_data",
+        store=True,
+        readonly=False,
+    )
+    reviewer_id = fields.Many2one(
+        comodel_name="res.users",
+        compute="_compute_definition_data",
+        store=True,
+        readonly=False,
+    )
     reviewer_group_id = fields.Many2one(
-        related="definition_id.reviewer_group_id", readonly=True
+        comodel_name="res.groups",
+        compute="_compute_definition_data",
+        store=True,
+        readonly=False,
     )
     reviewer_ids = fields.Many2many(
         string="Reviewers",
         comodel_name="res.users",
         compute="_compute_reviewer_ids",
         store=True,
     )
-    sequence = fields.Integer(string="Tier")
+    sequence = fields.Float(string="Tier")
```

Comment:
kittiu: change to float because escalated tier will be insertion, i.e., 1.1, 2.1 etc.